### DISCUSSION TITLE
be 1.1.2升级到1.1.3 查询报错

### DISCUSSION THREAD

**Discussion by SamuelBin** [apache/doris#13412] [A - General / Q&A] [answered]
be 1.1.2升级到1.1.3 查询报错

Caused by: java.sql.SQLException: (conn=312215) errCode = 2, detailMessage = Failed to get scan range, no queryable replica found in tablet: 29554

**Comment by LemonLiTree** [comment, marked as answer]
https://doris.apache.org/zh-CN/docs/dev/faq/sql-faq?_highlight=failed&_highlight=scan&_highlight=range&_highlight=queryable&_highlight=replica&_highlight=found&_highlight=tablet#q1-%E6%9F%A5%E8%AF%A2%E6%8A%A5%E9%94%99failed-to-get-scan-range-no-queryable-replica-found-in-tablet-xxxx
可以先看下文档说明，看下tablet=29554信息，还有问题可以进一步沟通

**Reply by SamuelBin** [reply]
可以了，不是doris本身升级的问题，迁移be配置没改好
